nsdb-cluster/src/main/scala/io/radicalbit/nsdb/cluster/index/MetadataIndex.scala

```diff
@@ -81,7 +82,12 @@ class MetadataIndex(override val directory: BaseDirectory) extends Index[Locatio
   }
 
   def getMetadata(metric: String): Seq[Location] = {
-    Try(query(_keyField, metric, Seq.empty, Integer.MAX_VALUE)(identity)) match {
+    val queryTerm = new TermQuery(new Term(_keyField, metric))
+
+    val reader                           = DirectoryReader.open(directory)
+    implicit val searcher: IndexSearcher = new IndexSearcher(reader)
```

Comment:
I'm not sure this belongs to this PR or not, but the canonical way to retrieve a searcher is by calling the superclass `Index` # `getSearcher` method